CoreScriptsRoot/CoreScripts/GamepadMenu.lua

```diff
@@ -31,6 +31,14 @@ local isTenFootInterface = tenFootInterface:IsEnabled()
 local radialButtons = {}
 local lastInputChangedCon = nil
 
+local D_PAD_BUTTONS =
```

Comment:
Keeping tack of input state may lead to some strange bugs, while less efficient, perhaps using http://wiki.roblox.com/index.php?title=API:Class/UserInputService/GetGamepadState may be a more solid solution?